<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UltraGrid settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Network">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QFormLayout" name="networkFormLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Base port</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="basePort">
           <property name="inputMask">
            <string/>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
           <property name="placeholderText">
            <string>5004</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Control port</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="controlPort">
           <property name="inputMask">
            <string/>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
           <property name="placeholderText">
            <string>8888</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="fecGroupBox">
         <property name="title">
          <string>Forward Error Correction</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>1</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QCheckBox" name="fecAutoCheck">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Auto</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="fecGroup_2" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="fecGroup">
               <property name="topMargin">
                <number>5</number>
               </property>
               <item>
                <widget class="QWidget" name="fecRadioGroup" native="true">
                 <layout class="QHBoxLayout" name="fecRadioLayout">
                  <item>
                   <widget class="QRadioButton" name="fecNoneRadio">
                    <property name="text">
                     <string>none</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="fecMultRadio">
                    <property name="text">
                     <string>mult</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="fecLdgmRadio">
                    <property name="text">
                     <string>LDGM</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="fecRsRadio">
                    <property name="text">
                     <string>RS</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QStackedWidget" name="stackedWidget">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="currentIndex">
                  <number>2</number>
                 </property>
                 <widget class="QWidget" name="mult">
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Multiplying factor</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="multSpin">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="value">
                      <number>2</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="lgdm">
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Max Loss</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="ldgmMaxLoss">
                     <property name="value">
                      <number>5</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                     <item>
                      <widget class="QRadioButton" name="ldgmSimpCpuRadio">
                       <property name="text">
                        <string>CPU</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="ldgmSimpGpuRadio">
                       <property name="text">
                        <string>GPU</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page">
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>k</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="ldgmK">
                     <property name="minimum">
                      <number>64</number>
                     </property>
                     <property name="maximum">
                      <number>8191</number>
                     </property>
                     <property name="value">
                      <number>256</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>m</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="ldgmM">
                     <property name="minimum">
                      <number>64</number>
                     </property>
                     <property name="maximum">
                      <number>8191</number>
                     </property>
                     <property name="value">
                      <number>192</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_12">
                     <property name="text">
                      <string>c</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="ldgmC">
                     <property name="minimum">
                      <number>2</number>
                     </property>
                     <property name="maximum">
                      <number>63</number>
                     </property>
                     <property name="value">
                      <number>5</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page_2">
                  <layout class="QHBoxLayout" name="horizontalLayout_10">
                   <item>
                    <widget class="QLabel" name="label_13">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>k</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="rsK">
                     <property name="maximum">
                      <number>255</number>
                     </property>
                     <property name="value">
                      <number>200</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_14">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>n</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="rsN">
                     <property name="maximum">
                      <number>255</number>
                     </property>
                     <property name="value">
                      <number>220</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Display">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="displayListView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentRow">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Gl</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SDL</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="displayStack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="glPage">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="glDeinterlace">
             <property name="text">
              <string>Deinterlace</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="glFullscreen">
             <property name="text">
              <string>Fullscreen</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="glNoDecorate">
             <property name="text">
              <string>Disable window decorations</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="glCursor">
             <property name="text">
              <string>Show cursor</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="glNoVsync">
             <property name="text">
              <string>Disable vsync</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="sdl_page">
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QCheckBox" name="sdlDeinterlace">
             <property name="text">
              <string>Deinterlace</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="sdlFullscreen">
             <property name="text">
              <string>Fullscreen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="accelTab">
      <attribute name="title">
       <string>Acceleration</string>
      </attribute>
      <layout class="QFormLayout" name="accelFormLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="decodeAccelCheck">
         <property name="text">
          <string>Video decode</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="accelHorizontalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>LDGM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ldgmCpuRadio">
           <property name="text">
            <string>CPU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ldgmGpuRadio">
           <property name="text">
            <string>GPU</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Advanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="errorsFatalBox">
         <property name="text">
          <string>Exit on all errors</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Encryption key</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="encryptionLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fecAutoCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>fecGroup_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fecNoneRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayListView</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>displayStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
